tests/Unit/Subscriber/Addons/CloudflareSubscriber/TestAutoPurgeByUrl.php

```diff
@@ -95,6 +97,8 @@ public function testShouldAutoPurgeByUrlWhenCacheEverything() {
 		Functions\when( 'current_user_can' )->justReturn( true );
 		Functions\when( 'get_current_user_id' )->justReturn( 1 );
 		Functions\when( 'is_wp_error' )->justReturn( false );
+		Functions\when( 'get_rocket_i18n_home_url' )->justReturn( 'http://local.com/' );
+		Functions\when( 'get_feed_link' )->justReturn( 'http://local.com/feed/' );
```

Comment:
We expect both of these functions to be invoked, ie both will run as part of this test. Let's change `when` (which means it may or may not run) to `expect` (meaning we "expect" it to run `once`, `with` these arguments, and then do something when done.